roles/wait_prometheus_crd/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: 'get prometheus crd'
```

Comment:
Would it make sense to add these at the end of `kube_prometheus` instead of creating a new role?

Reply:
I planed to add a `include_role` in kube_prometheus, but your idea is better